Mathlib/FieldTheory/SeparableClosure.lean

```diff
@@ -288,6 +296,7 @@ theorem sepDegree_eq_of_equiv (K : Type v) [Field K] [Algebra F K] (i : E ≃ₐ
 
 /-- The separable degree multiplied by the inseparable degree is equal
 to the (infinite) field extension degree. -/
+@[stacks 030L "Part 3"]
```

Comment:
I don't think this tag is right.

Reply:
Thanks. Again, this is stated in the paragraph right following 030L

Of course in characteristic 0 we have [E:F]=[E:F]s and [E:F]i=1. By multiplicativity (Lemma [09G9](https://stacks.math.columbia.edu/tag/09G9)) we have

[E:F]=[E:F]s[E:F]i
even in case some of these degrees are infinite. In fact, the separable degree and the inseparable degree are multiplicative too (see Lemma [09HK](https://stacks.math.columbia.edu/tag/09HK)).